roda-core/roda-core/src/main/java/org/roda/core/plugins/base/conversion/AbstractConvertPlugin.java

```diff
@@ -473,14 +642,182 @@ protected Report executeOnRepresentation(IndexService index, ModelService model,
+                          }
+                        } else {
+                          if (ignoreFiles) {
+                            reportItem.setPluginDetails("This file was ignored.");
+                          } else {
+                            pluginResultState = PluginState.FAILURE;
+                            reportItem.setPluginDetails(
+                              "This file was not ignored and it is not listed on the supported input file formats.");
+                          }
+                        }
+                      }
+
+                      if (!pluginResultState.equals(PluginState.SUCCESS)) {
+                        reportState = PluginState.FAILURE;
+                      }
+
+                    } catch (RuntimeException | NotFoundException | GenericException | RequestNotValidException
+                      | AuthorizationDeniedException | IOException | AlreadyExistsException e) {
+                      LOGGER.error("Error processing File {}: {}", file1.getId(), e.getMessage(), e);
+                      reportState = PluginState.FAILURE;
+                      reportItem.setPluginDetails(e.getMessage());
+                    } finally {
+                      if (file1.isReference() && tmpStorageService != null) {
+                        try {
+                          if (!job.getPluginType().equals(PluginType.INGEST)) {
+                            ModelUtils.removeTemporaryResourceShallow(job.getId(),
+                              ModelUtils.getAIPStoragePath(file1.getAipId()));
+                          }
+                        } catch (RequestNotValidException | IOException e) {
+                          LOGGER.error("Error on removing temporary AIP " + file1.getAipId(), e);
+                        }
+                      }
+                      reportItem.setPluginState(pluginResultState);
+                      report.addReport(reportItem);
+                      PluginHelper.updatePartialJobReport(this, model, reportItem, true, job);
+                    }
+
+                    boolean notifyEvent = true;
+                    createEvent(model, index, file1.getAipId(), file1.getRepresentationId(), file1.getPath(),
+                      file1.getId(), outputFormat, reportState, Arrays.asList(file1), newFiles, notifyEvent, job);
+
+                    if (!createDIP) {
+                      try {
+                        Representation rep = model.retrieveRepresentation(file1.getAipId(), newRepresentationID);
+                        createPremisSkeletonOnRepresentation(model, file1.getAipId(), rep);
+                      } catch (RequestNotValidException | GenericException | NotFoundException
+                        | AuthorizationDeniedException | ValidationException | IOException e) {
+                        LOGGER.error("Error running premis skeleton on new representation: {}", e.getMessage());
+                      }
+                    }
+                  }
```

Comment:
Call the "processFileShallow" method here.

Reply:
Done